todoist/types.py

```diff
@@ -67,7 +67,7 @@ def __init__(
             filter_items.append(f"@{label}")
 
         if assigned_self:
-            filter_items.append("(!(assigned to: others) & assigned)")
+            filter_items.append("(assigned to: me | !shared)")
```

Comment:
hmm, if there is a task in a shared project, is it the case that it should have an assignee? 

and if not who's responsibility is it

my understanding of this change is that it's indicating that rather than everyone's responsibility, it's nobodies

Reply:
that is correct, I think a task with no assignee and a due date shouldn't be everyone's responsibility, just set up wrong